<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleTab</class>
 <widget class="QWidget" name="VehicleTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="mYearLabel">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mIdLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="mLoadCapacityLabel">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="mModelLabel">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="mPassengersSb"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mBrandEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="mIdSb"/>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="mFuelConsumptionLabel">
     <property name="text">
      <string>Consumption</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="mYearCombo"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="mPassengersLabel">
     <property name="text">
      <string>Passengers Count</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QDoubleSpinBox" name="mFuelConsumptionSb">
     <property name="suffix">
      <string> l</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="mBrandLabel">
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QSpinBox" name="mLoadCapacitySb">
     <property name="suffix">
      <string> kg</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="mModelEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="mNewVehicleBtn">
     <property name="text">
      <string>New Vechicle</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QTableWidget" name="mVehicleTableWidget"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="mDelVehicleBtn">
     <property name="text">
      <string>Delete Vehicle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
